addons/event/models/event_event.py

```diff
@@ -229,6 +231,22 @@ def _compute_date_end_tz(self):
             else:
                 event.date_end_located = False
 
+    @api.depends('date_begin', 'date_end')
+    def _compute_is_ongoing(self):
+        for event in self:
+            event.is_ongoing = event.date_begin <= fields.Datetime.now() < event.date_end
+
+    def _search_is_ongoing(self, operator, value):
```

Comment:
Would be cool if you have some spare time to write at least one test for this one :) .